src/main/java/gov/cms/mat/cql_elm_translation/service/CqlTooling.java

```diff
@@ -109,11 +109,16 @@ protected CqlTranslator runTranslator(
   protected CqlTranslator processCqlData(RequestData requestData) {
     CqlTextParser cqlTextParser = new CqlTextParser(requestData.getCqlData());
     UsingProperties usingProperties = cqlTextParser.getUsing();
-    return TranslationResource.getInstance(
-            usingProperties != null
-                && ("FHIR".equals(usingProperties.getLibraryType())
-                    || "QICore".equals(usingProperties.getLibraryType())))
-        .buildTranslator(requestData);
+    boolean isFhir =
+        usingProperties != null
+            && ("FHIR".equals(usingProperties.getLibraryType())
+                || "QICore".equals(usingProperties.getLibraryType()));
+    // for now, we are treating QICore v7.0.0 the same as FHIR
+    if (isFhir && usingProperties.getVersion().equalsIgnoreCase("7.0.0")) {
```

Comment:
also, maybe this check can just see if the version is _at minimum_ `7.0.0`? I think 7.0.1 has already been released, and whenever we go to support STU8 it will also need to be treated the same way as STU7 (using the NpmModelInfoProvider)